common/src/thrift/file_system_master.thrift

```diff
@@ -42,6 +43,16 @@ struct MkdirTOptions {
   3: optional bool allowExists
 }
 
+struct PersistCommand {
+  1: common.CommandType commandType
```

Comment:
would this always be either `CommandType.Nothing` or `CommandType.Persist`? Could a boolean be used instead?